models/hh_cond_beta_gap_traub.h

```diff
@@ -87,13 +87,12 @@ The key differences between the current model and the model in [1]_ are:
 For details on asynchronicity in spike and firing events with Hodgkin Huxley models
 see :ref:`here <hh_details>`.
 
-See also [2]_.
```

Comment:
I would move this up, and rewrite the bullet list above, something like 

"... by Traub and Miles [1]_, with the following modifications:

- The model is a point neuron model.
- Following [2]_, the model only includes ... 
- Incoming spikes induce double-exponential conductance changes.
- The model incorporates gap junctions [X]_.
"

[X]_ should be a reference to

Hahne, J., Helias, M., Kunkel, S., Igarashi, J., Bolten, M., Frommer, A., & Diesmann, M. (2015). A unified framework for spiking and gap-junction interactions in distributed neuronal network simulations. Frontiers in Neuroinformatics, 9. https://doi.org/10.3389/fninf.2015.00022

Not sure where it ends up in the numbering.

Reply:
Added reference and text